src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -121,7 +122,11 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
       const found = this.validVersions.find((version: WorkflowVersion) => version.id === this.selectedVersion.id);
       this.isValidVersion = found ? true : false;
       this.downloadZipLink = Dockstore.API_URI + '/workflows/' + this.workflow.id + '/zip/' + this.currentVersion.id;
-      this.authors = [...this.selectedVersion.authors, ...this.selectedVersion.orcidAuthors];
```

Comment:
It's low odds, but the next line could either fail, e.g., due to flaky network, or be very slow, so I think you should clear the authors here so the previous ones aren't displayed.